frontend/src/index.css

```
@@ -585,33 +673,39 @@ textarea {
 .label {
   display: block;
   margin-bottom: 0.25rem;
-  color: #fff;
+  color: var(--aurral-text-muted);
   font-size: 0.875rem;
   font-weight: 500;
 }
 
 .volume-slider {
+  --volume-percent: 0%;
   -webkit-appearance: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)